src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
```

Comment:
It is a good practice to make utility classes `final` and add a private constructor to hide the implicit public one.